Prefetching-Library/README.md

```diff
@@ -123,3 +123,41 @@ After implementing a strategy, please consider sharing it by creating a PR to th
 To create a new build with the modifications implemented in the library, click on `Build > Make Project`.
```

Comment:
I have reformated some files, so I suggest hiding the whitespace changes when reviewing